### ISSUE TITLE
Smart Card / Windows Hello / Pin doesn't work

### ISSUE THREAD

**Issue by chazwoza** [AzureAD/azure-activedirectory-library-for-cordova#67]
I'm using smart card authentication and windows hello PIN authentication but neither works. When using ADAL with Cordova, windows doesn't pop up the smart card window.

**Comment by sgrebnov** [comment]
Sounds like some missing capabilities (sharedUserCertificates), could you please try adding the following setting in config.xml and then build/run the app to see if this helps
```
<preference name="adal-use-corporate-network" value="true" />
```

This should add the following capabilities:

```
      <Capability Name="enterpriseAuthentication" />
      <Capability Name="privateNetworkClientServer" />
      <Capability Name="sharedUserCertificates" />
```

**Comment by chazwoza** [comment]
Hi Sgrebnov, unfortunately adding the preference didn't work... is there somewhere I can add the capabilities directly?

**Comment by sgrebnov** [comment]
Yes, after adding `windows` platform you can open `<your app>/platforms/windows/CordovaApp.sln` in Visual Studio, then double click on `package.windows10.appxmanifest` (I think you are targeting windows10) and then select `Capabilities` tab.

![image](https://cloud.githubusercontent.com/assets/981580/15249729/cbebe3b2-1929-11e6-9fc4-e9008e7f1c60.png)


After that you can run Windows Phone 8.1 project directly from Visual Studio, see [Windows Platform Guide](https://cordova.apache.org/docs/en/6.x/guide/platforms/win8/index.html) for more details.

**Comment by sgrebnov** [comment]
@chazwoza did you get a chance to verify required capabilities? What plugin version are you using - I see that there was some issue with missing capabilities on Windows 10 fixed in 0.7.0.

**Comment by chazwoza** [comment]
Tried and it doesn't work - can't figure out how to install v0.7.0. Running "ionic plugin add cordova-plugin-ms-adal" only added v 0.6.2

**Comment by daserge** [comment]
Can you try `cordova plugin add cordova-plugin-ms-adal`? 
It should take the plugin from [npm](https://www.npmjs.com/package/cordova-plugin-ms-adal), and the latest published version now is 0.7.1.

**Comment by sgrebnov** [comment]
Closing as there is no response, please re-open if this is still actual.
